### PR TITLE
Fixed icon and updated dependencies

### PR DESCRIPTION
**Pull request by felixoi**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -91,9 +91,10 @@ lazy val discourse = project.settings(
     "io.circe"                   %% "circe-core"           % circeVersion,
     "io.circe"                   %% "circe-generic-extras" % circeVersion,
     "io.circe"                   %% "circe-parser"         % circeVersion,
+    "com.typesafe.akka"          %% "akka-http"            % akkaHttpVersion,
```
Comment (Katrix) [inline build.sbt:94]: Why did you add this dep?

**Review by Katrix — COMMENTED** [review]
Did you make sure all akka deps are the same?

File: build.sbt (hunk 1 above)
Comment (felixoi) [reply build.sbt:94]: Because I had to force `akka-http-circe` to use this version of akka-http. Otherwise he get conflicting akka versions.

**Review by Katrix — APPROVED** [review]
